esp-hal-common/Cargo.toml

```diff
@@ -134,7 +144,7 @@ debug = [
     "esp32s3?/impl-register-debug",
 ]
 
-log = ["dep:log"]
+# Enable support for `dmt`, for `esp-hal-common` and also for all our dependencies
```

Comment:
```suggestion
# Enable support for `defmt`, for `esp-hal-common` and also for all our dependencies
```

I don't think I've ever heard of someone refer to defmt as dmt. Is this a typo by any chance?

Reply:
lol what do you think